src/kernel/src/fs/dev/mod.rs

```diff
@@ -326,7 +324,7 @@ pub fn mount(
     cred: &Arc<Ucred>,
     path: VPathBuf,
     parent: Option<Arc<Vnode>>,
-    opts: HashMap<String, Box<dyn Any>>,
+    _opts: MountOpts,
```

Comment:
Better to use `_` here because `devfs` does not support options. Not sure why I keep it here.